OpenRA.Mods.Common/Lint/CheckDeathTypes.cs

```diff
@@ -51,7 +55,7 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Ruleset ru
 							continue;
 
 						if (!warhead.DamageTypes.Overlaps(deathTypes))
-							emitError("Actor type `{0}` does not define a death animation for weapon `{1}`!"
+							emitError("Actor type `{0}` does neither define a death animation nor spawns an actor on death for weapon `{1}`!"
```

Comment:
Weird english here.  "Actor type `{0}` doesn't define a death animation or spawn an actor on death for weapon `{1}`!" would be better.